README.md

```diff
@@ -146,3 +146,22 @@ Script: [rm-ds-store](rm-ds-store)
 ```sh
 rm-ds-store
 ```
+### ⏲  Get time at a location
+
+Script: [clock](clock)
+Dependencies: Linux: [timedatectl](https://www.freedesktop.org/software/systemd/man/timedatectl.html) Mac: sudo
+
+```sh
+clock list
+clock <Time Zone>
+```
+
+### 🌧 Get weather forecast
+
+Script: [weather](weather)
+Dependencies: [curl](https://curl.haxx.se/)
```

Comment:
I think `curl` is pre-installed on most of the system. Am I wrong?
If no, maybe we can remove it to keep the README clean.

Reply:
Maybe I should update the PR template to `...mentioned any outstanding dependencies...`.